compliance-checker/meta.yaml

```diff
@@ -26,7 +22,7 @@ requirements:
         - requests
         - python-dateutil
     run:
-        - python >=2.7,<3
```

Comment:
But I thought that with the new modification we should constraint the python version at the `build` filed only.